xml/ay_custom_user_scripts.xml

```diff
@@ -484,12 +489,13 @@ echo "Testing the pre script" &gt; /tmp/pre-script_out.txt
          <para>
           If this is <literal>true</literal>, every single line of a shell
           script is logged. Perl scripts are run with warnings turned on.
+          For other interpreters use directly parameter in interpreter field.
```

Comment:
> For other interpreters use directly parameter in interpreter field.

What parameter? What field? Suffixed onto the name of the binary of the language?

Reply:
parameter depends on interpreter. e.g. -x for bash, -w for ruby, etc.
By field I should probably use parameter, but it is confusing with script parameter. Maybe I should talk about XML element?